llvm/lib/Target/RISCV/RISCVFrameLowering.cpp

```diff
@@ -548,12 +480,7 @@ void RISCVFrameLowering::allocateAndProbeStackForRVV(
 
   if (EmitCFI) {
     // Set the CFA register back to SP.
-    unsigned Reg = STI.getRegisterInfo()->getDwarfRegNum(SPReg, true);
-    unsigned CFIIndex =
-        MF.addFrameInst(MCCFIInstruction::createDefCfaRegister(nullptr, Reg));
-    BuildMI(MBB, MBBI, DL, TII->get(TargetOpcode::CFI_INSTRUCTION))
-        .addCFIIndex(CFIIndex)
-        .setMIFlags(MachineInstr::FrameSetup);
+    CFIBuilder.buildDefCFARegister(SPReg);
```

Comment:
Isn't this a behaviour change? Before it was MachineInstr::FrameSetup, now it's MachineInstr::FrameDestroy

Reply:
Thank you for spotting this!
I tried to be careful, but apparently not careful enough.